doc/classes/SkeletonProfileHumanoid.xml

```diff
@@ -5,6 +5,63 @@
 	</brief_description>
 	<description>
 		A [SkeletonProfile] as a preset that is optimized for the human form. This exists for standardization, so all parameters are read-only.
+		A humanoid skeleton profile is structured as follows:
+		[codeblock]
+		Root
+		└ Hips
+		│  ├─ LeftUpperLeg
```

Comment:
Hips -> Spine I think

Reply:
It seems like it is attached to the Root bone
https://github.com/godotengine/godot/blob/7827c8e37010241b376748b22db85847c8614dbd/scene/resources/skeleton_profile.cpp#L474-L475